challenge_2/python/CindyB/src/singleNumber.py

```diff
@@ -0,0 +1,8 @@
+def findSingleNumber(myArray):
+    for x in myArray:
+        count = sum(map(lambda s: s == x,myArray))
```

Comment:
You're totally right. This is indeed some nested loops and so O(n^2). Thanks for pointing this out. I'm gonna check out for Counter class or defaultdict to get an idea of the O(n) solution.